src/test/java/com/neo4j/docker/TestPluginInstallation.java

```diff
@@ -60,13 +61,7 @@
     @BeforeAll
     public static void checkVersionIsCompatibleWithTest()
```

Comment:
do we still need this method?

Reply:
hopefully not!
I'll tidy it up once this is definitely sorted out